src/m_menu.c

```diff
@@ -3187,11 +3184,6 @@ setup_menu_t weap_settings1[] =  // Weapons Settings screen
   {"Enable Fist/Chainsaw\n& SG/SSG toggle", S_YESNO|S_BOOM, m_null, M_X,
    M_Y+ weap1_toggle*M_SPC, {"doom_weapon_toggles"}},
 
-  {"", S_SKIP, m_null, M_X, M_Y + weap1_stub2*M_SPC},
```

Comment:
I find the former menu item even more obscure. I like this one as a little curiosity.